web/src/pages/Settings/Content.tsx

```diff
@@ -1,28 +1,28 @@
-import {Tabs} from 'antd';
+// import {Tabs} from 'antd';
 import DataStore from 'components/Settings/DataStore';
 import {useDataStoreConfig} from 'providers/DataStoreConfig/DataStoreConfig.provider';
 import * as S from './Settings.styled';
 
-const TabsKeys = {
+/* const TabsKeys = {
   DataStore: 'dataStore',
-};
+}; */
 
 const Content = () => {
   const {dataStoreConfig} = useDataStoreConfig();
 
   return (
     <S.Container>
       <S.Header>
-        <S.Title>Settings</S.Title>
+        <S.Title>Configure Data Store</S.Title>
       </S.Header>
 
-      <S.TabsContainer>
+      {/* <S.TabsContainer>
         <Tabs size="small">
-          <Tabs.TabPane key={TabsKeys.DataStore} tab="Configure Data Store">
```

Comment:
I guess you did this so we can go back to tabs more easily once we need it right?

Reply:
That's right. I think we will add more options soon.

Reply:
I personally would leave the tab... does not distract me. It is, however, @olha23 UX.

Reply:
@olha23 for final approval 😄

Reply:
we will add tabs when there is more than one

Reply:
Sold! feel free to merge @jorgeepc